docs/learn/assets.md

```diff
@@ -99,7 +99,148 @@ To ensure the authenticity of the images, please generate the hash of the upload
 
 </details>
 
-When you metadata file is ready, upload it (centralised or decentralised storage).
+Such JSON can be generated manually or using tools, such as [`@lukso/lsp-utils`](https://github.com/lukso-network/lsp-utils). To generate the JSON from predefined fields:
+
+- Install the `@lukso/lsp-utils` library
+
+```bash
+npm i @lukso/lsp-utils
+```
+
+- Then import the `generateLSP4JSON` to generate the JSON
+
+```js
+import { generateLSP4JSON } from '@lukso/lsp-utils';
```

Comment:
I can't find the docs for this function in:

https://docs.lukso.tech/tools/lsp-utils/LSP4DigitalAssetMetadata

Do you think it is missing? @b00ste 

It would be nice to put a link to the page that explains the function

Reply:
yeah, need to sync docs to latest version, we just released on monday